containers/podman-desktop
# docs: Document how to write, publish and install Podman Desktop extensions

### What does this PR do?
How to write, publish and install a Podman Desktop extension

### Screenshot/screencast of this PR

Click on the netlify PR check to browse the result

### What issues does this PR fix or reference?

Fixes #1651 

### How to test this PR?
Click on the netlify PR check to browse the result

Change-Id: I0317b0badc0bb08cc10f06bff78a85f4445cbe6d
